docs/examples/ex22.py

```diff
@@ -50,14 +50,14 @@ def edge_jump(w):
         n = w.n
         dw1 = grad(w['u1'])
         dw2 = grad(w['u2'])
-        return h * ((dw1[0] - dw2[0]) * n[0] +\
+        return h * ((dw1[0] - dw2[0]) * n[0] +
                     (dw1[1] - dw2[1]) * n[1]) ** 2
 
     eta_E = edge_jump.elemental(fbasis[0], **w)
     
     tmp = np.zeros(m.facets.shape[1])
     np.add.at(tmp, fbasis[0].find, eta_E)
-    eta_E = np.sum(0.5*tmp[m.t2f], axis=0)
+    eta_E = np.sum(.5 * tmp[m.t2f], axis=0)
```

Comment:
This little change reminds me that this biggish collection of changes leading to the next release might be a reasonable opportunity to adopt black, should that be something that's wanted eventually.  As discussed previously, I don't have a strong opinion on it; but it seems to be popular and does have some advantages, such as new contributors not having to learn the house style.

Reply:
I'm using it in some of the smaller projects and automatically checking black-conformity in CI. I'll consider using it here aswell.

Reply:
There are some choices in black that I don't like and I'd probably want to fine tune the config quite a bit so I'll postpone it. Let's open another issue for that I think.